docs/guides/update-guide/810-to-820.md

```diff
@@ -27,3 +27,11 @@ This may change the performance characteristics. If you have tuned your system f
 
 - It is recommended to follow a sequential update path when updating to version 8.2. For example, if running on version 8.0, first update to 8.1, then update to 8.2.
 - Migration of data during the version 8.2 update could take longer than previous versions, especially for datasets containing a large amount of incidents data.
+
+## Tasklist
+
+- New configuration is required to enable Tasklist Process Page:
+  - `CAMUNDA_TASKLIST_IDENTITY_BASEURL` environment variable should be configured to your Identity host
+  - `CAMUNDA_TASKLIST_IDENTITY_RESOURCE_PERMISSIONS_ENABLED` environment variable should be set to true or false (default is false) and configures Tasklist to use Authorizations coming from Identity.
```

Comment:
Are these the names that are used directly by Tasklist or in the help charts? For the flag I would omit Identity, so `CAMUNDA_TASKLIST_RESOURCE_PERMISSIONS_ENABLED` instead of `CAMUNDA_TASKLIST_IDENTITY_RESOURCE_PERMISSIONS_ENABLED` since resource permission or resource authorizations is a new concept globally in C8 and not just related to Identity, but it's not a "hard" opinion from my side.

Reply:
Those are env vars. On the name, I added the IDENTITY as all other IDENTITY configurations are working like that in tasklist.